lib/bower-plugin.js

```diff
@@ -53,19 +54,20 @@ function BowerWebpackPlugin(options) {
   this.manifestFiles = opt.manifestFiles ? [].concat(opt.manifestFiles) : ["bower.json"];
   this.includes = opt.includes ? [].concat(opt.includes) : [/.*/];
   this.excludes = opt.excludes ? [].concat(opt.excludes) : [];
+  this.searchResolveModulesDirectories = opt.searchResolveModulesDirectories === false ? false : true;
```

Comment:
The flag is false by default, contrary to what the JSDoc says. Sorry! Will clarify the JSDoc.